ports/qt5-base/cmake/qt_fix_makefile_install.cmake

```diff
@@ -1,34 +1,33 @@
 #Could probably be a vcpkg_fix_makefile_install for other ports?
 function(qt_fix_makefile_install BUILD_DIR)
     #Fix the installation location
-    file(TO_NATIVE_PATH "${CURRENT_INSTALLED_DIR}" NATIVE_INSTALLED_DIR)
-    file(TO_NATIVE_PATH "${CURRENT_PACKAGES_DIR}" NATIVE_PACKAGES_DIR)
-    
-    if(WIN32)
-        string(SUBSTRING "${NATIVE_INSTALLED_DIR}" 2 -1 INSTALLED_DIR_WITHOUT_DRIVE)
-        string(SUBSTRING "${NATIVE_PACKAGES_DIR}" 2 -1 PACKAGES_DIR_WITHOUT_DRIVE)
+    set(MSYS_HACK "")
+    if(VCPKG_HOST_IS_WINDOWS)
+        file(TO_NATIVE_PATH "${CURRENT_INSTALLED_DIR}" NATIVE_INSTALLED_DIR)
+        file(TO_NATIVE_PATH "${CURRENT_PACKAGES_DIR}" NATIVE_PACKAGES_DIR)

```

Comment:
Probably this should be outside the `if(VCPKG_HOST_IS_WINDOWS)`?

Reply:
The conversion is pointless outside `VCPKG_HOST_IS_WINDOWS`. But I also didn't replace `NATIVE` with `CURRENT` in the other code path, so this broken, too.